audformat/core/database.py

```diff
@@ -289,6 +290,47 @@ def drop_tables(
         for table_id in table_ids:
             self.tables.pop(table_id)
 
+    def files_duration(
+            self,
+            files: typing.Union[str, typing.Sequence[str]],
+    ) -> typing.Union[pd.Timedelta, pd.Series]:
```

Comment:
I think it is not a good idea to return different things for a single file or several files. I would change to always return a series.

Reply:
Changed it accordingly.